.github/PULL_REQUEST_TEMPLATES/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### Merge Checklist
+
+<!-- Complete all tasks below to make sure that we can merge your pull request. -->
+
+- [ ] I have read the [contribution guide](doc/contributing.md)
+- [ ] I have added my info to [copying.md](copying.md) (only first time contributors)
+- [ ] I have run `make checkall` and fixed all mentioned problems
```

Comment:
this will show up in every pull request as a 3-point checklist. not sure we want that. maybe have those in a comment so they will hopefully be read, but they don't show up in every pull request text box?

Reply:
I would say this is the entire point of having templates :D First time contributors should ideally see this every time they view their PR, instead of just on creation (and then promptly forget it). And we can say "plz do the stuff in the checklist" instead of "do this and do that".

Regular contributors like us could just delete the entire thing if they want to anyway...

Reply:
Then how about adding "If you are an experienced contributor, you may just delete these checkpoints"? :)

Reply:
Ok I'm fine with that!

Done